templates/zerver/help/preview-your-message-before-sending.md

```diff
@@ -1,11 +1,15 @@
 # Previewing your message
 
-You can preview your message before you send it by clicking the **eye** (![eye](/static/images/help/eye.png)) icon.
+You can preview your message before you send it by clicking the **eye**
+(![eye](/static/images/help/eye.png)) icon at the bottom of the **Compose message box**.
 
 ![Original content](/static/images/help/before-preview.png)
 
+This is bring up the message preview:
+
 ![Previewed](/static/images/help/during-preview.png)
```

Comment:
You can delete this line as well as the image from the `static/images/help` folder.